ospo-book/chapters/01-chapter.md

```diff
@@ -18,3 +24,117 @@ The book is structured in a user-friendly and practical manner, with a focus on
+
+### Understand existing and desired open source adoption
+
+Once an organization has assessed the level of open source used, contributed, or produced in the organization and why establishing an OSPO can help an organization manage the risks and opportunities of what open source, open works and collaboration brings, and ensure that its open source activities are effectively managed and aligned with the organization's strategic goals and objectives. 
+
+While this is a book about Open Source Programs Offices (OSPOs), it is important to note that establishing an OSPO might not the starting point for open source operations. Before establishing an OSPO (and keep reading the content of the book), companies and organizations need to assess their current goals and relationship with using and collaborating to open source software projects. 
+
+Below, people will find a checklist to assess and better understand their possible current stage and potential next steps.
+
+Assessing open source adoption is critical because it sets the foundation for successful open source operations. Without proper understanding and adoption of open source, an OSPO may not be effective in achieving the desired outcomes. 
+
+* **☑️ Open Source Software (or open works) Usage:** Evaluate the level of open source software usage within your organization. Are there any specific open source projects that are widely used? Are there any projects that are critical to the organization's operations?
+
+* **☑️ Knowledge and Understanding of Open Source:** Evaluate the level of knowledge and understanding of open source within your organization. Are the different actors that will be or are currently involved in open source familiar with open source licensing models and requirements? Do they understand the benefits and risks of using open source software?
+
+* **☑️ Culture:** Evaluate the culture within your organization to determine if it is conducive to open source operations. Is there a culture of collaboration and sharing? Are the different actors that will be or are currently involved in open source willing to contribute to open source projects?
+
+* **☑️ Tools and Processes:** Evaluate the tools and processes in place to support open source operations. Are there any existing tools or processes that can be leveraged for open source operations? Are there any gaps in tools or processes that need to be addressed?
+
+* **☑️ Addressing Gaps:** Determine if there are any gaps in open source adoption or readiness and develop a plan to address them. This may include training those actors that will be or are currently involved in open source on open source software usage and licensing, developing new tools and processes to support open source operations, or establishing an OSPO to coordinate open source activities.
+
+* **☑️ Overall, gather input from stakeholders on these areas** by asking the folowing questions
```

Comment:
One question that I think is missing is "how much open-source software is already being used in the organisation?" My experience is that asking this question will help your stakeholder to get more aware of the crucial role open source already plays in the organization. Most stakeholders are unaware of how much open source is used in their organization and the risks that it is not yet professionally managed.